src/create-element.js

```diff
@@ -75,11 +75,13 @@ export function createVNode(type, props, key, ref, original) {
 		// a _nextDom that has been set to `null`
 		_nextDom: undefined,
 		_component: null,
-		constructor: undefined,
-		_original: original
+		constructor: undefined
 	};
 
-	if (original == null) vnode._original = vnode;
+	Object.defineProperty(vnode, '_original', {
```

Comment:
I'd love a quick comment in the code about the reason for using `Object.defineProperty` here. For people new to our codebase this may be a helpful thing to do :slightly_smiling_face:

Reply:
Will do, good shout Marvin